javascript/javascript2/week1/spiritanimal.js

```diff
@@ -1,28 +1,31 @@
-const spiritAnimas = [ "fullmoonwolf","The Butterfly","Teen wolf","Spidersense"]
+const spiritAnimals = [
+  "fullmoonwolf",
+  "The Butterfly",
+  "Teen wolf",
+  "Spidersense",
+];
 const button1 = document.getElementById("btn1");
 
- const randomNumber1 = Math.floor(Math.random()*10); 
-    
- const h2 = document.getElementById("spiritinfo");
+const randomNumber1 = Math.floor(Math.random() * 4);
 
-const getSpiritAnimal = function(){
-   
-    const userName = document.getElementById("name").value;        
-    h2.innerHTML = userName + '-'+ spiritAnimals[randomNumber1];               
-}
-button1.addEventListener('click', getSpiritAnimal);
+const h2 = document.getElementById("spiritinfo");
+
+const getSpiritAnimal = function () {
+  const userName = document.getElementById("name").value;
+  h2.innerHTML = userName + "-" + spiritAnimals[randomNumber1];
+};
+button1.addEventListener("click", getSpiritAnimal);
 
-const button2 = document.getElementById("btn2") ;
-button2.innerHTML= "Get new spirit animal";
+const button2 = document.getElementById("btn2");
+button2.innerHTML = "Get new spirit animal";
 
-    
-function getNewSpiritAnimal(){
-    const userName = document.getElementById("name").value; 
-    if(userName){
-        const randomNumber2 = Math.floor(Math.random()*10); 
-    h2.innerHTML = userName + '-'+ spiritAnimals[randomNumber2];
-}else{
+function getNewSpiritAnimal() {
+  const userName = document.getElementById("name").value;
+  if (userName) {
+    const randomNumber2 = Math.floor(Math.random() * 10);
```

Comment:
10 is hardcoded in randomNumber2 , need to be changed like in randomNumber1 above